### PR TITLE
LPS-115771 - Update date string format for date-fns

### PR DESCRIPTION
**Pull request by bryceosterhaus**

### CODE REVIEW

**Comment by bryceosterhaus** [comment]
ci:test:relevant

**Comment by bryceosterhaus** [comment]
ci:test:format

**Comment by jbalsas** [comment]
> LocalFile.FragmentsUI#EditFragment

Seems unrelated. Throws an `ORMException`.

> LocalFile.FragmentsImporter#CannotImportNonZip

This is surprising... fails a locator looking for `Toast.js`???? `WaitForElementPresent(locator1 = "//script[contains(@src,'ClayToast.js')]");`... I'll investigate...

From the ones in common with upstream... some have to do with `alerts`... smoke from our `Alert` or `Notification` message perhaps, same as the one above about `ClayToast`...

> test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8

This is likely a snapshot :(

**Comment by jbalsas** [comment]
FYI @bryceosterhaus, please try to follow our lovely [code review guidelines](https://github.com/liferay/liferay-frontend-guidelines/blob/master/dxp/code_review.md#guidelines) specially the part about `Always provide a description` and provide meaningful titles and descriptions for all of us to enjoy! 🤗

File: modules/yarn.lock [hunk 1]
```diff
@@ -1081,6 +1081,15 @@
     "@clayui/layout" "^3.0.1"
     classnames "^2.2.6"
 
+"@clayui/alert@3.3.4":
```
Comment (jbalsas) [inline modules/yarn.lock:1084]: Not sure this is deduped correctly... 

> 369 changes: 129 additions & 240 deletions

This is usually a warning sign for me, uneven additions and deletions... in this case, surprisingly we deduped more than added, but there's a mix of new `@clayui` packages not deduped and other dependencies down the line `camelcase` that are in fact now better than they were... 🤔

**Comment by jbalsas** [comment]
So, turns out `headless-discovery-web` has the following suppression:

> source.formatter.excludes=\
    modules/apps/headless/headless-discovery/headless-discovery-web/package.json

Which means that clay package versions did not get updated via SF, which is causing the duplicated packages... 

Going to manually change for now and open a communication line with @nhpatt to see if we can loosen up that suppression a bit.

**Comment by jbalsas** [comment]
Hey @bryceosterhaus, I can't push to your origin, so I had to send a new PR to https://github.com/liferay-frontend/liferay-portal/pull/38

I think it would be a good thing for everyone to provide edit permissions to their `liferay-portal` forks to reviewers so we can easily do minor tweaks and get PRs moving without having to create new ones.

**Pull request closed without merging**